rero_ils/alembic/8145a7cdef99_reindex_items_with_invalid_ean.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019-2023 RERO
+# Copyright (C) 2019-2023 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""reindex items with invalid EAN."""
+
+from logging import getLogger
+
+from elasticsearch_dsl.query import Q
+
+from rero_ils.modules.documents.api import DocumentsIndexer, DocumentsSearch
+
+# revision identifiers, used by Alembic.
+revision = '8145a7cdef99'
+down_revision = '5f0b086e4b82'
+branch_labels = ()
+depends_on = None
+
+LOGGER = getLogger('alembic')
+indexing_chunck_size = 100
+
+
+def upgrade():
+    """Upgrade database."""
+    query = DocumentsSearch().filter(
```

Comment:
Difficult to say if we need to do it using invenio or if this can be performed by elasticsearch....